src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -236,8 +236,9 @@ class TypeSpecTest {
         .addFun(FunSpec.builder("fooBar")
             .addModifiers(KModifier.PUBLIC, KModifier.ABSTRACT)
             .addAnnotation(AnnotationSpec.builder(headers)
-                .addMember("value", "%S", "Accept: application/json")
-                .addMember("value", "%S", "User-Agent: foobar")
+                .addMemberArrayElement("value", Array<String>::class, "%S",
```

Comment:
Right. We don't really need the generic type to generate code though, so passing `Array<Any>::class` should always be fine.